### PR TITLE
Fix key error, update training configuration

### PR DESCRIPTION
**Pull request by matt-gardner**
This is actually a very small change, but it depends on #947.  @rajasagashe, this should fix the key errors we were seeing.  It also has an updated training config, with optimizer parameters that match those in the original paper.  I had more luck with using a learning rate of 0.01, though - the 0.1 run I did crashed with an error I'm still looking into.

### CODE REVIEW

**Comment by rajasagashe** [comment]
Thanks! Any reason why in the "question_token_indexers" field of the config file, "lowercase_tokens" is no longer set to true?

**Comment by matt-gardner** [comment]
The question and table are both now hard-coded to always be lowercased (because it sure looks like that's what SEMPRE does - the questions in the training data are always lowercase).  So that flag isn't necessary.
